.markdownlint.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Should be removed.

Reply:
Commit to remove this file has been pushedn now